### PR TITLE
UG Formatting cs2101

### PR DESCRIPTION
**Pull request by lingshanng**
- cropped some pictures to fit
- changed `page-break-after` to `page-break-before` so there's less space before the Headers
- add links to find command

### CODE REVIEW

**Review by eeliana — APPROVED** [review]
I think it is fine. The tables breaking off probably no choice

**Review by Xiaoyunnn — APPROVED** [review]
I think its fine too

File: docs/UserGuide.md [hunk 1]
```diff
@@ -96,7 +103,7 @@ Section | Representation
 ---------|---------
 Menu Bar | The area where you can click to navigate and view some of TAB's features (e.g. help, reminder, and calendar). See [Menu Bar Shortcuts](#menu-bar-shortcuts) for more details.
 Menu Item | A button that you can click to view the stated feature, or to quit TAB (for <kbd>Exit</kbd>).
```
Comment (angkoonhwee) [inline docs/UserGuide.md:105]: I think you missed out this one

File: docs/UserGuide.md [hunk 2]
```diff
@@ -208,7 +216,7 @@ This command pops open a help window that includes a summary of our commands, as
 
 Format: `help`
 
-- You can select a cell and copy its value using <kbd>Ctrl</kbd> + <kbd>C</kbd> on Windows or <kbd>COMMAND</kbd> + <kbd>C</kbd> on Mac.
+- You can select a cell and copy its value using <kbd>CTRL</kbd> + <kbd>C</kbd> on Windows or <kbd>COMMAND</kbd> + <kbd>C</kbd> on Mac.
 - You can click <kbd>Copy URL</kbd> button to copy the link to this user guide.
```
Comment (angkoonhwee) [inline docs/UserGuide.md:220]: eh not sure if this on needs to be changed also since it is 2 words

**Review by angkoonhwee — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (lingshanng) [reply docs/UserGuide.md:105]: This is a button on the UI so i didn't change

**Review by angkoonhwee — APPROVED** [review]

**Pull request merged**